app/models/ability.rb

```diff
@@ -64,10 +64,16 @@ def curator_permissions
     curation_concern_read_permissions
   end
 
-  # Abilities that should be granted to patron
+  # Abilities that should be granted to patrons in an authorized group
+  # Ability to read authenticated visibility items is handled by #user_groups
+  def music_patron_permissions
```

Comment:
Is there some way to move these site-specifics out of the model and into configuration?